x/mint/keeper/msg_server.go

```diff
@@ -27,7 +27,10 @@ func NewMsgServerImpl(k Keeper) types.MsgServer {
 // UpdateParams updates the params.
 func (ms msgServer) UpdateParams(goCtx context.Context, msg *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
 	if ms.authority != msg.Authority {
-		return nil, errors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", ms.authority, msg.Authority)
+		// this has been duplicated from the x/gov module to prevent a cyclic dependency
+		// see: https://github.com/cosmos/cosmos-sdk/blob/91d14c04accdd5ded86888514401f1cdd0949eb2/x/gov/types/errors.go#L20
+		duplicatedGovErr := fmt.Errorf("expected gov account as only signer for proposal message")
```

Comment:
Can we change the error? https://github.com/cosmos/cosmos-sdk/pull/16567#discussion_r1230557555
Possibly we should be consistent and define an error in mint directly that has the same name.